servo/connectors/kubernetes.py

```diff
@@ -1901,7 +1916,7 @@ async def raise_for_pod_status(self):
             fmt_str = ", ".join(pod_fmts)
             raise servo.AdjustmentRejectedError(
                 message=f"{len(unschedulable_pods)} pod(s) could not be scheduled for deployment {self.name}: {fmt_str}",
```

Comment:
elsewhere we don't use `message` as a named argument but as positional; I suggest doing it here as well for consistency